src/components/ui4/scenes/HomeSceneUi4.tsx

```diff
@@ -0,0 +1,201 @@
+import { SectionView } from '../SectionView'
+
+interface Props extends EdgeSceneProps<'home'> {}
+
+const TEMP_PADDING_REM = 0.5 // To be built-in to SceneWrapper when fully UI4
+
+const DUMMY_BLOG_DATA = [
+  {
+    localeTitle: {
+      en: 'Latest Edge News 1',
+      es: 'Que es Thorchain'
+    },
+    localeBody: {
+      en: 'My blog body text\nlorem ipsum',
+      es: 'El texto principal de mi blog'
+    },
+    localeBlogUrl: {
+      en: 'https://edge.app/whatisthorchain',
+      es: 'https://edge.app/whatisthorchain?lang=es'
+    },
+    lightImageUrl: 'https://edge.app/images/tc-dark.png',
+    darkImageUrl: 'https://nyc3.digitaloceanspaces.com/edgecontent/UI4/testBlogImg.png'
+  },
+  {
+    localeTitle: {
+      en: 'Latest Edge News 2',
+      es: 'Que es Thorchain'
+    },
+    localeBody: {
+      en: 'My blog body text\nlorem ipsum',
+      es: 'El texto principal de mi blog'
+    },
+    localeBlogUrl: {
+      en: 'https://edge.app/whatisthorchain',
+      es: 'https://edge.app/whatisthorchain?lang=es'
+    },
+    lightImageUrl: 'https://edge.app/images/tc-dark.png',
+    darkImageUrl: 'https://nyc3.digitaloceanspaces.com/edgecontent/UI4/testBlogImg.png'
+  },
+  {
+    localeTitle: {
+      en: 'Latest Edge News 3',
+      es: 'Que es Thorchain'
+    },
+    localeBody: {
+      en: 'My blog body text\nlorem ipsum',
+      es: 'El texto principal de mi blog'
+    },
+    localeBlogUrl: {
+      en: 'https://edge.app/whatisthorchain',
+      es: 'https://edge.app/whatisthorchain?lang=es'
+    },
+    lightImageUrl: 'https://edge.app/images/tc-dark.png',
+    darkImageUrl: 'https://nyc3.digitaloceanspaces.com/edgecontent/UI4/testBlogImg.png'
+  }
+]
+
+export const HomeSceneUi4 = (props: Props) => {
+  const { navigation } = props
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
+  const { width: screenWidth } = useSafeAreaFrame()
+  const needsPasswordCheck = useSelector(state => state.ui.passwordReminder.needsPasswordCheck)
+
+  const handleBuyPress = useHandler(() => {
+    navigation.navigate('buyTab', {})
+  })
+  const handleSellPress = useHandler(() => {
+    navigation.navigate('sellTab', {})
+  })
+  const handleFioPress = useHandler(() => {
+    navigation.navigate('fioAddressList', {})
+  })
+  const handleSwapPress = useHandler(() => {
+    navigation.navigate('exchangeTab', {})
+  })
+
+  // TODO: Replace dummy data with info server data
+  const blogData = DUMMY_BLOG_DATA
+
+  // Show the password reminder on mount if required:
+  useAsyncEffect(
+    async () => {
+      if (needsPasswordCheck && !isMaestro()) {
+        await Airship.show(bridge => <PasswordReminderModal bridge={bridge} navigation={navigation} />)
+      }
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
+  )
+
+  return (
+    <NotificationSceneWrapper navigation={navigation} scroll hasTabs padding={theme.rem(TEMP_PADDING_REM)}>
+      <SectionView wideSpacing>
+        <>
```

Comment:
Do these fragments do anything? It seems like `{/* */}` comments or even blank lines could work just as well to divide the sections.

Reply:
Yes, the fragments are telling the `SectionView` which children should be displayed together as a single section group. In this section, we have 3 children in one section.

Reply:
Oooh! That's really neat.